Example/.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-"presets": ["react-native"]
-}
\ No newline at end of file
+  "presets": ["react-native"]
```

Comment:
This whitespace change was `react-native-git-upgrade`, and I can undo if desired